src/test/java/com/epam/edumanagementsystem/teacher/impl/TeacherServiceImplTest.java

```diff
@@ -84,4 +91,42 @@ void canCreateTeacher() {
     void canNotCreateTeacher() {
         assertThrows(ObjectIsNull.class, () -> teacherService.create(null));
     }
+
+    @Test
+    @DisplayName("Update Teacher positive case - given all parameters")
+    void testUpdateFields() {
+        Optional<Teacher> ofResult = Optional.of(teacher);
+
+        when(teacherRepository.save(org.mockito.Mockito.any())).thenReturn(theTeacherToBeChanged);
+        when(teacherRepository.findById(org.mockito.Mockito.any())).thenReturn(ofResult);
+
+        when(userService.findByEmail(org.mockito.Mockito.any())).thenReturn(user);
+        TeacherDto actualUpdateFieldsResult = teacherService.updateFields(new TeacherDto());
+        assertEquals("testEmail@example.org", actualUpdateFieldsResult.getEmail());
+        assertEquals("Surname", actualUpdateFieldsResult.getSurname());
+        assertEquals("Name", actualUpdateFieldsResult.getName());
+        verify(teacherRepository).findById(org.mockito.Mockito.any());
+    }
```

Comment:
Why don't you import Mockito package?

Reply:
Because "any" my intellij understand the other lib, but if I give just any the object will be null